### ISSUE TITLE
Cannot update Spyder

### ISSUE THREAD

**Issue by canoetraveler** [spyder-ide/spyder#12635]
### What steps will reproduce the problem?

Running Windows 7, service pack 1 on system with 64-bit Intel i7-6600U CPU, 8GB of memory.

I was running Spyder 4.0.1 and attempted to update to 4.1.2 by issuing cmd: conda install spyder=4.1.2 
On the next reboot, Anaconda and Sypder would not start. I tried several fixes: conda update --all, conda update sypder, conda install sypder=4.0.1. All failed. Then I uninstalled Anaconda and reinstalled Anaconda3. Now I cannot install any updates to Spyder above 3.3.6. Looking for options to get back to Spyder 4.       

## Versions

* Spyder version: 3.3.6 
* Python version: 3.7.4
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
pyflakes >=0.6.0  :  2.1.1 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.4.2 (OK)
sphinx >=0.6.6    :  2.2.0 (OK)
rope >=0.9.4      :  0.14.0 (OK)
jedi >=0.9.0      :  0.15.1 (OK)
nbconvert >=4.0   :  5.6.0 (OK)
pandas >=0.13.1   :  0.25.1 (OK)
numpy >=1.7       :  1.16.5 (OK)
sympy >=0.7.3     :  1.4 (OK)
cython >=0.21     :  0.29.13 (OK)
qtconsole >=4.2.0 :  4.5.5 (OK)
IPython >=4.0     :  7.8.0 (OK)
matplotlib >=2.0.0:  3.1.1 (OK)
pylint >=0.25     :  2.4.2 (OK)
```

**Comment by dalthviz** [comment]
Hi @canoetraveler sorry to hear that. What version of conda are you using?

Maybe you can create a conda env. You can create it and activate it using something like:
```cmd
conda create -n spyder4 python=3.7 spyder=4.1.2
conda activate spyder4
```
Let us know if creating the env helps :)

**Comment by dalthviz** [comment]
Also @canoetraveler are you using the latest Anaconda installer (`Anaconda3-2020.02-Windows-x86_64`) ?

**Comment by canoetraveler** [comment]
OK. Thank you. The above instructions worked to get Spyder 4.1.2 running. But now I don't have access to any of the libraries like numpy, Matplotlib, panda, etc.  Can you give me a bit more guidance?

Here is the conda version information: 
(spyder4) C:\Users\mhagen>conda info
active environment : spyder4
active env location : C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4
            shell level : 2
user config file : C:\Users\mhagen\.condarc
populated config files : C:\Users\mhagen\.condarc
conda version : 4.8.3
conda-build version : 3.18.9
python version : 3.7.4.final.0
virtual packages :
       base environment : C:\Users\mhagen\AppData\Local\Continuum\anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\mhagen\AppData\Local\Continuum\anaconda3\pkgs
                          C:\Users\mhagen\.conda\pkgs
                          C:\Users\mhagen\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs
                          C:\Users\mhagen\.conda\envs
                          C:\Users\mhagen\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.8.3 requests/2.22.0 CPython/3.7.4 Windows/7 Windows/6.1.7601
          administrator : False
             netrc file : None
           offline mode : False

**Comment by canoetraveler** [comment]
One other thing. This is a laptop PC without a ton of storage left on the SSD drive. Once we get this environment set up and configured properly, can we delete the old libraries? That is, can you point to instructions to manage Python/Anaconda environments to conserve disk space?

**Comment by ccordoba12** [comment]
@canoetraveler, if you want to update Spyder in your base environment, you need to run

    conda update anaconda
    conda install spyder=4.1.2

Please try those commands to see if they work for you.

**Comment by canoetraveler** [comment]
@ccordoba12 , With all due respect, those are the commands that got me in a mess in the first place.  Don't I need to update this new environment?

BTW, after rebooting, I now get the following errors:

  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\widgets\debugging.py", line 143, in set_spyder_breakpoints
    CONF.get('run', 'breakpoints', {}))
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder_kernels\comms\commbase.py", line 538, in __call__
    call_dict, call_data, self._comm_id)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 130, in _get_call_return_value
    with self.comm_channel_manager(comm_id, queue_message=not interrupt):
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 87, in comm_channel_manager
    timeout)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 168, in _wait
    raise TimeoutError(timeout_msg)
TimeoutError: Timeout while waiting for comm port.


A message asks me to install these two modules. But when I try I get conflict errors.
![image](https://user-images.githubusercontent.com/30637705/81299707-3e8bac80-903c-11ea-9a1f-7b1ff8c1cfbf.png)

Is the reason I'm having so much difficulty because this PC is running Windows 7?

**Comment by ccordoba12** [comment]
> Don't I need to update this new environment?

No, you don't need to.

> TimeoutError: Timeout while waiting for comm port.

This is a common error on Windows. We're working to fix it.

> A message asks me to install these two modules. But when I try I get conflict errors.

How are you trying to update those packages? And what are the error?

**Comment by canoetraveler** [comment]
I tried installing the packages as: conda install jedi=0.15.2. 
It would discover conflicts and try to recover for, like 30 minutes, then fail.

Anyway. I kind'a punted. I uninstalled both Anaconda and Python 3.7. (I had a long Zoom meeting to attend.) Then reinstalled the latest Windows-64 Anaconda3 from https://www.anaconda.com/products/individual. Now I'm back to where I was two days ago. I'm at Spyder 4.0.1 and I have working libraries/packages. I think I'll stay here on this PC for a while. 
Thank you _very_ much for your assistance.
